tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -133,6 +133,7 @@ jobs:
 - job: Build_NNAPI_EP
   pool: Linux-CPU-2019
   timeoutInMinutes: 30
+  condition: notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI')
```

Comment:
Why? So if I manually queue a build, it would be different than what CI does?

Reply:
It's to avoid BUILD_NNAPI_EP and NNAPI_EP_MASTER start together.
So far, I'm not familiar with these variables. What's the suggestion.